.github/workflows/static_checks.yml

```diff
@@ -205,6 +207,16 @@ jobs:
         run: |
           bazel run //scripts:string_resource_validation_check -- $(pwd)
 
+      - name: Binary files check
```

Comment:
**Self-reminder:** Add if (!cancelled) check upon approval.

Reply:
Why not do that now?

Reply:
I missed adding that earlier, but it's included now.

_Also resolving since - done_